lib/spack/spack/test/install.py

```diff
@@ -436,16 +436,6 @@ def test_pkg_install_paths(install_mockery):
     shutil.rmtree(log_dir)
 
 
-def test_log_install_without_build_files(install_mockery):
-    """Test the installer log function when no build files are present."""
-    # Get a basic concrete spec for the trivial install package.
-    spec = Spec('trivial-install-test-package').concretized()
-
-    # Attempt installing log without the build log file
-    with pytest.raises(IOError, match="No such file or directory"):
-        spack.installer.log(spec.package)
```

Comment:
Why is this test removed?

Reply:
I believe this specific test was designed to raise an error when nothing was installed with the `install` function, but now that it uses globs, it is possible to have no matching files and there is no error. We could add an assert to keep the error if you want.

Reply:
I think we should keep it. It looks like we were relying on that error to ensure that the build logs exist at install time. I'm fine with an error from llnl.util.filesystem.install or an error in spack.installer.log as long as we preserve that check somehow.